src/v/storage/api.h

```diff
@@ -24,6 +26,8 @@ class api {
       , _log_conf(std::move(log_conf)) {}
 
     ss::future<> start() {
+        finjector::shard_local_badger().register_probe(
+          fs_failure_probe::name(), &fs_finject::get_probe());
```

Comment:
@dotnwat is this a decent place to register/deregister the failpoints? Looking for suggestions.